samples/AutoButton.js

```diff
@@ -20,20 +20,12 @@ var delay = 10;
 
 // import modules
 var gpio = require("gpio");
-var pins = require("arduino101_pins");
 
 // listen for button presses on this input
-var input = gpio.open({
-    pin: pins.IO11,
-    direction: 'in',
-    edge: 'rising'
-});
+var input = gpio.open({pin: "IO7", mode: 'in', edge: 'rising'});
 
 // simulate button presses on this output
-var output = gpio.open({
-    pin: pins.IO12,
-    direction: 'out'
-});
+var output = gpio.open({pin: "IO10"});
```

Comment:
Yeah, I kind of varied the usage throughout the samples to perhaps highlight the different options.